Gemfile.lock

```diff
@@ -1,26 +1,26 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.6)
+    activesupport (5.1.3)
```

Comment:
I think this is because of the dependencies. But running 'bundle exec jekyll serve' should have worked for you to watch the output. Anyway, for this PR just remove this file from teh commit and send a PR with blog post file only.